boswatch/inputSource/lineInInput.py

```diff
@@ -40,23 +40,7 @@ def _runThread(self, dataQueue, lineInConfig, decoderConfig):
             lineInProc.setStderr(open(paths.LOG_PATH + "asla.log", "a"))
             lineInProc.start()
 
-            mmProc = ProcessManager(str(lineInConfig.get("mmPath", default="multimon-ng")), textMode=True)
-            if decoderConfig.get("fms", default=0):
-                mmProc.addArgument("-a FMSFSK")
-            if decoderConfig.get("zvei", default=0):
-                mmProc.addArgument("-a ZVEI1")
-            if decoderConfig.get("poc512", default=0):
-                mmProc.addArgument("-a POCSAG512")
-            if decoderConfig.get("poc1200", default=0):
-                mmProc.addArgument("-a POCSAG1200")
-            if decoderConfig.get("poc2400", default=0):
-                mmProc.addArgument("-a POCSAG2400")
-            if lineInConfig.get("mmChar"):
-                mmProc.addArgument("-C " + str(lineInConfig.get("mmChar")))
-            mmProc.addArgument("-f alpha")
-            mmProc.addArgument("-t raw -")
-            mmProc.setStdin(lineInProc.stdout)
-            mmProc.setStderr(open(paths.LOG_PATH + "multimon-ng.log", "a"))
+            mmProc = self.startmm(decoderConfig, lineInProc.stdout)
             mmProc.start()
```

Comment:
Das sollte auch noch mit in die Funktion - die heißt ja immerhin startmm daher erwarte ich auch, dass er dann gestartet returned wird - rein von der Logik her

Reply:
Da bin ich auch schon drüber gestolpert, wusste dann aber nicht wie das elegant zusammen mit dem `mmProc.Stop()` kombinierbar wäre. Irgendwie fände ich es auch komisch, den start im Base zu haben aber den Stop nicht.
Vorschläge?
`startmm()` umbenennen?

Reply:
Da bin ich auch schon drüber gestolpert, hatte aber keine gute Idee wie man das elegant lösen könnte:
Das `mmProc.stop()` ist ja auch noch da, und ich fände es nicht ganz ideal, wenn das start ins Base wandert, aber der stop immer noch im Modul ist.
Vorschläge?
`startmm()` umbenennen?

Reply:
in `.getDecoderInstance` ?

Reply:
Wenn wir es so belassen, würde ich noch die Doku dazu anpassen...

Reply:
Jo ich denke, mit dem Stand können wir jetzt auf jeden Fall ins Rennen gehen